emulators/vpcs/Portfile

```diff
@@ -0,0 +1,51 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:filetype=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+
+name                vpcs
+version             0.8c
+revision            126
```

Comment:
The `revision` starts at 0.

Reply:
Here is where you could create the `release` variable. 

`set release     126`

Reply:
Yes - lastic - the revision here means the Portfile revision, so that's 0. You were thinking of the SVN revision, and that's a different thing.

Reply:
How do I create this variable 'release' ?

Reply:
just like mf2k said -- "set release 126" -- tcl thankfully is really easy to learn !

Reply:
Facepalm, I was using ${release} before setting it, lesson learned.